logging/bin/change_internal_password.sh

```diff
@@ -184,123 +164,107 @@ if [[ $response == 4* ]]; then
+             log_notice " NOTE: *NEVER* change the password for the 'admin' account from within the  "
+             log_notice " Kibana web-interface.  The 'admin' password should *ONLY* be changed via   "
+             log_notice " this script (logging/bin/change_internal_password.sh)                      "
+             echo ""
+          fi
+          ;;
+        logcollector)
           echo ""
-          log_notice "+============================================================================+"
-          log_notice "|                    *********** IMPORTANT NOTE ***********                  |"
-          log_notice "|                                                                            |"
-          log_notice "| The password for the Elasticsearch 'admin' user was changed as requested.  |"
-          log_notice "|                                                                            |"
-          log_notice "| Since a new password for the 'admin' user was NOT provided, one was        |"
-          log_notice "| auto-generated for the account. The generated password is shown below.     |"
-          log_notice "|                                                                            |"
-          log_notice "| Generated 'admin' password:  $NEW_PASSWD                                        |"
-          log_notice "|                                                                            |"
-          log_notice "| You can change the password for the 'admin' account at any time, by        |"
-          log_notice "| re-running this script.                                                    |"
-          log_notice "|                                                                            |"
-          log_notice "+============================================================================+"
-          log_notice "| NOTE: *NEVER* change the password for the 'admin' account from within the  |"
-          log_notice "| Kibana web-interface.  The 'admin' password should *ONLY* be changed via   |"
-          log_notice "| this script (logging/bin/change_internal_password.sh)                      |"
-          log_notice "+============================================================================+"
+          log_notice "                    *********** IMPORTANT NOTE ***********                   "
+          log_notice "                                                                             "
+          log_notice " After changing the password for the [logcollector] user, you should restart "
+          log_notice " the Fluent Bit pods to ensure log collection is not interrupted.            "
+          log_notice "                                                                             "
+          log_notice " This can be done by submitting the following command:                       "
+          log_notice "         kubectl -n $LOG_NS delete pods -l 'app=fluent-bit, fbout=es'        "
```

Comment:
I thought we were moving away from fixing the width with spaces and trying to center text

Reply:
The notice text was existing code...I removed the box characters that surrounded the notices but didn't bother removing the leading spaces.  I can do that if you think it is worthwhile.

Reply:
Probably not worth it as part of this PR. Maybe it's something we can address separately.